draft-ietf-rats-endorsements.md

```diff
@@ -371,7 +371,7 @@ trust in the Verifier it will use.  In such a case, the Relying Party may have
 a constrained Verifier embedded in it that is only capable of appraising Evidence
 provided by its desired Verifier.  Thus, the Relying Party uses its embedded Verifier
 for purposes of appraising its desired Verifier which it treats as only an Attester,
-and once verified, then uses it for verification of all other Attesters.
+and once verified via appraisal, then uses it for appraisal of all other Attesters.
```

Comment:
I find _"verified via appraisal"_ unnecessarily heavy.

Let's do _"appraised"_, or leave it as-is.

(The former seems more in line with the PR's spirit.)

Reply:
okay